tests/e2e/test-build-js.R

```diff
@@ -7,10 +7,7 @@ cat(
   file = index_js_path
 )
 rhino::build_js()
-testthat::expect_identical(
-  readLines(min_js_path),
-  readLines(fs::path("..", "test-files", "sayHello-app-min-js.txt"))
-)
+testthat::expect_true(readLines(min_js_path) != "var App;App={};")

```

Comment:
Instead of comparing the output to a static file that may change in the future, this checks if the built js file is not a result of an empty `index.js`. The main test to see if `build_js()` work should be in the final cypress test.